### PR TITLE
Feature/entregable 2

### PR DESCRIPTION
**Pull request by jveracal553**
Subimos los cambios hechos en el feature/entregable2. Se ha culminado con el mockup en html y css.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,28 +1,217 @@
...
         <button>Cerca a ti</button>
         <button>
-          <img src="/img/soccer.svg" alt="" class="soccer" />
+          <img src="/img/soccer.svg" alt="  " class="soccer" />
         </button>
       </form>
-    </nav>
-  </body>
-</html>
+    </nav> -->
```
Comment (Drleugim) [inline index.html]: Please remove commented code

File: style.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.search {
+/* .search {
```
Comment (Drleugim) [inline style.css]: Please remove commented code

**Comment by Drleugim** [comment]
Please remove all commented code in all files.

**Comment by jveracal553** [comment]
> Please remove all commented code in all files.

Ok @Drleugim, comments were removed from the file index.html and style.css

File: style.css [hunk 3]
```diff
@@ -1,14 +1,234 @@
...
+  height: 14vw;
+  margin: 1px;
+  width: 24.8%;
+  margin-bottom: 30px;
+}
+
+.img-cancha {
+  margin-top: 25px;
+  align-items: center;
+  /* border: 2px solid rgb(222, 92, 92); */
```
Comment (Drleugim) [inline style.css]: On this file I still see commented code. Please remove.

File: style.css [hunk 4]
```diff
@@ -1,14 +1,234 @@
...
+  }
+  .topnav input[type="text"] {
+    border: 1px solid #ccc;
+  }
+}
+
+.slider-holder {
+  width: 730px;
+  height: 400px;
+  /* background-color: yellow; */
```
Comment (Drleugim) [inline style.css]: There's still commented code on this file. Please remove

**Review by Drleugim — APPROVED** [review]
LGTM

**Pull request merged**